frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/cater-capacity/cater-capacity.component.html

```diff
@@ -0,0 +1,29 @@
+<div *ngIf="canCaterFor" style="max-width: 25vw">
+    <span>{{
+        type === 'material' ? 'Dieses Material ' : 'Diese Einsatzkraft '
+    }}</span>
+
+    <span *ngIf="(canCaterFor | caterCapacityCount) === 0; else caterInfo"
+        >kann keinen Patienten helfen.</span
+    >
+    <ng-template #caterInfo>
+        <span>{{
+            canCaterFor.logicalOperator === 'or'
+                ? 'kann nur Patienten einer Verletzungsschwere gleichzeitig helfen.'
```

Comment:
```suggestion
                ? 'kann nur Patienten eines Verletzungsgrads gleichzeitig versorgen.'
```

Reply:
Ob Verletzungsgrad das bessere Wort ist, weiß ich nicht @christianzoellner was sagst du?

Reply:
I think `Sichtungskategorie` might be the correct word for that, I'll create a commit to apply your suggestions.

Reply:
Just asked Christian about it, he agreed to `Sichtungskategorie`.